src/main/java/it/gov/pagopa/pu/bff/security/JwtAuthenticationFilter.java

```diff
@@ -50,6 +51,7 @@ protected void doFilterInternal(@NonNull HttpServletRequest request,@NonNull Htt
         UsernamePasswordAuthenticationToken authToken = new UsernamePasswordAuthenticationToken(userInfo, null, authorities);
```

Comment:
let's store input token into Security context
```suggestion
        UsernamePasswordAuthenticationToken authToken = new UsernamePasswordAuthenticationToken(userInfo, token, authorities);
```